app/models/manageiq/providers/redfish/physical_infra_manager/operations/firmware.rb

```diff
@@ -0,0 +1,55 @@
+module ManageIQ::Providers::Redfish
+  module PhysicalInfraManager::Operations::Firmware
+    def update_firmware_async(firmware_binary, servers)
+      validate_update_firmware(firmware_binary, servers)
+
+      with_provider_connection do |client|
+        update_service = client.UpdateService
+        raise MiqException::MiqFirmwareUpdateError, 'UpdateService is not enabled' unless update_service
+
+        protocol, url = compatible_firmware_url(update_service, firmware_binary)
+
+        response = update_service.post(
+          :path    => update_service.Actions['#UpdateService.SimpleUpdate'].target,
```

Comment:
You can write this as `response = update_service.Actions["#UpdateService.SimpleUpdate"].post(:field => "target", ...)`.

Reply:
Sweet